Tests/Realm.Tests/Database/InstanceTests.cs

```diff
@@ -763,7 +763,7 @@ public void GetInstance_WhenDynamicAndDoesntExist_ReturnsEmptySchema()
         [TestCase("søren")]
         public void GetInstance_WhenPathContainsNonASCIICharacters_ShouldWork(string path)
         {
```

Comment:
this is only for internal testing so the message might be a bit verbose, but still it addresses the potential issue.. similar issue for the other test files In this PR